engine/src/main/scala/cromwell/engine/backend/local/SharedFileSystemBackend.scala

```diff
@@ -23,31 +21,41 @@ import scala.util.{Failure, Success, Try}
 object SharedFileSystemBackend {
   type LocalizationStrategy = (String, Path, WorkflowDescriptor) => Try[Unit]
 
-  private def localizeFromGcs(originalPath: String, executionPath: Path, descriptor: WorkflowDescriptor): Try[Unit] = Try {
+  private[local] def localizeFromGcs(originalPath: String, executionPath: Path,
```

Comment:
Just FYI, In the new LocalBackend I made `originalPath` be a `Path` too, so we don't have to have this special case for GCS, it'll be built in the path already.